### DISCUSSION TITLE
Secondary and tertiary lists

### DISCUSSION THREAD

**Discussion by JoelMTaylor** [retypeapp/retype#335] [Q&A] [answered]
I have run into an odd problem. Normally, when I have secondary and tertiary lists, I indent them with a tab in the ```.md``` document. This has been working for me well up to this point. However, unexpectedly, I have one page that decides to not display properly. 

In \edumate\Enrolments\Student\Student.md, I have the following code:
![image](https://user-images.githubusercontent.com/97919641/170629284-fa04953a-6270-4faf-96a0-1ed9b8d56c0f.png)

This is displaying as: 
![image](https://user-images.githubusercontent.com/97919641/170629325-64d28730-e3c5-4c70-a2e3-96cbaf036c45.png)

My main question: what is the preferred method for creating secondary and tertiary lists? I have seen pages say two space, four spaces, a tab, etc. Is there a preferred method for Retype?

If this continues to display incorrectly (and a tab is preferred/fine), I might change it to two spaces in this instance (I normally frown upon inconsistency in documents - when there is no reason). However, if there is a solution, that is nice, too.

Thank you for your ongoing assistance.

**Comment by geoffreymcgill** [comment, marked as answer]
Hi @JoelMTaylor. Retype tries to follow the CommonMark spec as closely as possible and I believe nested listed should be indented by either 2 or 4 spaces. I don't think the spec supports using Tab chars for indentation of list items. I could be wrong but will need to dig through the [spec](https://spec.commonmark.org/0.28/#lists) to confirm.

[Example](https://spec.commonmark.org/dingus/?text=%23%23%20Try%20CommonMark%0A%0A1.%20**Starts%20On**%20-%20try%20and%20select%20a%20time%0A%20%20%20%20%2B%20Click%20into%20field%2C%20**select%20the%20time%20first**%0A%20%20%20%20%2B%20Select%20on%20**%20date%20to%20start**%0A2.%20**End%20On**%20-%20never%20(it%20will%20only))

```md
## Try CommonMark

1. **Starts On** - try and select a time
    + Click into field, **select the time first**
    + Select on ** date to start**
2. **End On** - never (it will only)
```

Maybe some of your samples were using 4-spaces and others were using 1-tab char? The spec supports nesting with 2-spaces as well.

Hope this helps.

**Reply by JoelMTaylor** [reply]
I did try and find the common standard for Markdown. However, due to my ignorant searches, I was not able to do so with much confidence. I will use the resources provided more in the future.

Thank you.

**Reply by geoffreymcgill** [reply]
No worries. We need to make reference to the CommonMark spec and some of the online Markdown playground editors in our docs.

**Reply by JoelMTaylor** [reply]
So I changed it (the example given) to four spaces instead of a tab. However, it is coming up with the same issue. 

A tab on one line and four spaces on the other did seem to work (if that helps at all).

**Reply by JoelMTaylor** [reply]
I had a look at the spec you linked. It seems to say that only lists of the same type (list OR ordered) is supported. Would you understand that to be true from your reading of it?

Note, I have a mixed list - numbers then "+".

@geoffreymcgill, what do you think?
